scripts/vnet_route_check.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import json
+from swsssdk import ConfigDBConnector
+from swsssdk import SonicV2Connector
+
+
+def get_vnet_intfs():
+    config_db = ConfigDBConnector()
+    config_db.connect('CONFIG_DB')
+
+    intfs_data = config_db.get_table("INTERFACE")
+    vlan_intfs_data = config_db.get_table("VLAN_INTERFACE")
+
+    vnet_intfs = {}
+    for k, v in intfs_data.items():
+        if 'vnet_name' in v:
+            vnet_name = v['vnet_name']
+            if vnet_name in vnet_intfs:
+                vnet_intfs[vnet_name].append(k)
+            else:
+                vnet_intfs[vnet_name] = [k]
+
+    for k, v in vlan_intfs_data.items():
+        if 'vnet_name' in v:
+            vnet_name = v['vnet_name']
+            if vnet_name in vnet_intfs:
+                vnet_intfs[vnet_name].append(k)
+            else:
+                vnet_intfs[vnet_name] = [k]
+
+    return vnet_intfs
+
+
+def get_rifs_oid():
```

Comment:
are you expecting to get the rifs from the counter db? why not from the config db?
if you would like to get the counters per rif you should rename the function name